### PR TITLE
br: hide_encryption_scope

### PR DESCRIPTION
**Pull request by Leavrth**
### What problem does this PR solve?

Issue Number: ref #44199 

Problem Summary:

### What is changed and how it works?
the feature server-side encryption won't be GA, so hide the parameter.
### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: br/pkg/storage/azblob.go [hunk 1]
```diff
@@ -77,13 +77,17 @@ func defineAzblobFlags(flags *pflag.FlagSet) {
...
+	_ = flags.MarkHidden(azblobEncryptionScope)
 }
 
 func hiddenAzblobFlags(flags *pflag.FlagSet) {
 	_ = flags.MarkHidden(azblobEndpointOption)
 	_ = flags.MarkHidden(azblobAccessTierOption)
 	_ = flags.MarkHidden(azblobAccountName)
 	_ = flags.MarkHidden(azblobAccountKey)
+	_ = flags.MarkHidden(azblobSASToken)
+	//_ = flags.MarkHidden(azblobEncryptionScope)
```
Comment (YuJuncen) [inline br/pkg/storage/azblob.go:90]: Please add a comment about why this should be moved out...

Comment (Leavrth) [reply br/pkg/storage/azblob.go]: remove the comment, just mark hidden twice

**Review by Ehco1996 — APPROVED** [review]
lgtm

**Review by YuJuncen — APPROVED** [review]

**Pull request merged**